internal/plugin.go

```diff
@@ -61,12 +61,13 @@ type Plugin struct {
 }
 
 var defaults = types.PolicyDefaults{
-	Categories:        []string{"CM Configuration Management"},
-	ComplianceType:    "musthave",
-	Controls:          []string{"CM-2 Baseline Configuration"},
-	RemediationAction: "inform",
-	Severity:          "low",
-	Standards:         []string{"NIST SP 800-53"},
+	Categories:             []string{"CM Configuration Management"},
+	ComplianceType:         "musthave",
+	MetadataComplianceType: "",
```

Comment:
Here based the logic here https://github.com/stolostron/config-policy-controller/blob/main/controllers/configurationpolicy_controller.go#L1845, default MetadataComplianceType need to be set as `""` rather than `"musthave"`

Reply:
This defaults to an empty string in Go, so this line is unnecessary.